frontend/server/src/main/java/com/amazonaws/ml/mms/wlm/ModelManager.java

```diff
@@ -41,13 +42,15 @@
     private ConfigManager configManager;
     private WorkLoadManager wlm;
     private ConcurrentHashMap<String, Model> models;
+    private ConcurrentLinkedDeque<Model> startupModels;
```

Comment:
ArrayList<Model> should be enough, this is only added during startup time.

Reply:
True. I agree.